test/test_functionalization.py

```diff
@@ -461,14 +461,11 @@ def f(x, y):
         with capture_logs() as logs:
             y = f(x1_not_functional, x2_functional)
 
-        # I think the alias trace is coming from the fact that x2 is technically *not*
-        # a LoggingTensor (instead it *contains* a LoggingTensor), but x1 *is* a LoggingTensor.
-        # The important thing here though is that functionalization ran the "+" kernel
+        # Make sure that functionalization ran the "+" kernel
         # with a functional + non-functional tensor, and wrapped the output appropriately.
         self.assertExpectedInline('\n'.join(logs), """\
 $2 = torch._ops.aten.add.Tensor($0, $1)
-$3 = torch._ops.aten.alias_copy.default($2)
```

Comment:
@bdhirsh I'm guessing the reason this went away is because when `__torch_function__` wraps the output of add, it calls `as_subclass` which calls into `alias`, and now that we disable `__torch_function__` for logging tensor it no longer does that.

Reply:
That sounds right, and the new trace looks better!